src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -2,21 +2,23 @@
 
 import bio.terra.cda.app.configuration.ApplicationConfiguration;
 import bio.terra.cda.app.service.QueryService;
+import bio.terra.cda.app.util.CDAUtils;
 import bio.terra.cda.app.util.QueryTranslator;
 import bio.terra.cda.generated.controller.QueryApi;
 import bio.terra.cda.generated.model.Query;
 import bio.terra.cda.generated.model.QueryResponseData;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Objects;
+import java.util.*;
```

Comment:
My IDE collapsed this,  I don't typically like .* inclusions.

Reply:
In intellij, the setting is in `Editor > Code Style > Java > Imports > Class count to use import with '*'`. Setting this to 1000 should disable this behavior.